index.css

```diff
@@ -323,7 +350,7 @@ g.edge.selected > polygon {
 #drop_target {
   background-color: #ddd;
   border: dashed 2px #888;
-  padding: 1em;
+  padding: 4em 1em;
```

Comment:
Taller drop target. Ideally the whole page should be a target though, while this would just be a non-functional "invite" to drop.